### ISSUE TITLE
version 1.1.0+1 build failed

### ISSUE THREAD

**Issue by qinho** [pikaju/flutter-braintree#51]
update to v1.1.0+1, build the app, it happened:

FAILURE: Build failed with an exception.

* What went wrong:
Could not determine the dependencies of task ':app:lintVitalRelease'.
> Could not resolve all artifacts for configuration ':app:debugRuntimeClasspath'.
   > Could not resolve org.jfrog.cardinalcommerce.gradle:cardinalmobilesdk:2.2.4-1.
     Required by:
         project :app > project :flutter_braintree > com.braintreepayments.api:drop-in:5.0.1 > com.braintreepayments.api:three-d-s
ecure:3.14.2
      > Could not resolve org.jfrog.cardinalcommerce.gradle:cardinalmobilesdk:2.2.4-1.
         > Could not get resource 'https://maven.google.com/org/jfrog/cardinalcommerce/gradle/cardinalmobilesdk/2.2.4-1/cardinalmo
bilesdk-2.2.4-1.pom'.
            > Could not HEAD 'https://maven.google.com/org/jfrog/cardinalcommerce/gradle/cardinalmobilesdk/2.2.4-1/cardinalmobiles
dk-2.2.4-1.pom'.
               > Connect to maven.google.com:443 [maven.google.com/172.217.160.110] failed: Connection timed out: connect
      > Could not resolve org.jfrog.cardinalcommerce.gradle:cardinalmobilesdk:2.2.4-1.
         > Could not get resource 'https://storage.googleapis.com/download.flutter.io/org/jfrog/cardinalcommerce/gradle/cardinalmo
bilesdk/2.2.4-1/cardinalmobilesdk-2.2.4-1.pom'.
            > Could not HEAD 'https://storage.googleapis.com/download.flutter.io/org/jfrog/cardinalcommerce/gradle/cardinalmobiles
dk/2.2.4-1/cardinalmobilesdk-2.2.4-1.pom'.
               > Connection reset

**Comment by dinit-bwd** [comment]
I am facing same issue. Any update?

**Comment by Akash9151** [comment]
I'm also facing same issue. Any Update....?

**Comment by DriftingShadows** [comment]
same here, my builds failing. @pikaju any ideas?

**Comment by JoyEnergiser** [comment]
Me too please.

**Comment by samaralii** [comment]
same here. any fix yet?

**Comment by VNBijani** [comment]
sane here, any update?

**Comment by TejaDroid** [comment]
With using below code found working.
you just have to add to your app level build.gradle the depenency like so:

dependencies {
   .....
    implementation ('com.braintreepayments.api:drop-in:5.0.1')
}

and to you android level build.gradle the mave repository like so:

allprojects {
    repositories {
        google()
        jcenter()
        maven {
            url "https://cardinalcommerce.bintray.com/android"
            credentials {
                username 'braintree-team-sdk@cardinalcommerce'
                password '220cc9476025679c4e5c843666c27d97cfb0f951'
            }
        }
    }
}

**Comment by muiz6** [comment]
I fixed it by adding this:
```groovy
android {
        lintOptions {
        // ...
        checkReleaseBuilds false // Insert this line
    }
}
```
to app level [buid.gradle](https://developer.android.com/studio/build) file

**Comment by ilicmilan** [comment]
Any update on this?

**Comment by pikaju** [comment]
Sorry for the wait guys, this should hopefully be fixed in version `1.2.1` of `flutter_braintree`.
